dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -65,6 +66,24 @@ window.Dashboard = {
 		Marbles.history.navigate('login'+ redirectPath);
 	},
 
+	__isCertInstalled: function() {
+		var self = this, dispatchIfCertIsMissing = function(a, b) {
+			if (a + b !== 200 + 200) {
```

Comment:
the only thing we can do is check if we're on `https://dashboard.FLYNN_CLUSTER` and assume, that the cert isn't working when `https://controller.FLYNN_CLUSTER` returns `0`. But it might be a timeout or something else..